widgets/Dataplex-batchsink.json

```diff
@@ -0,0 +1,465 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name": "Dataplex",
+  "configuration-groups": [
+    {
+      "label": "Basic",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName",
+          "widget-attributes": {
+            "placeholder": "Name used to identify this sink for lineage"
+          }
+        }
+      ]
+    },
+    {
+      "label": "Connection",
+      "properties": [
+        {
+          "widget-type": "hidden",
+          "label": "Connection",
+          "name": "connection",
+          "widget-attributes": {
+            "connectionType": "Dataplex"
```

Comment:
why do we have this widget?

Reply:
This widget is pointing out to GCPConnectorConfig class, as we are reusing it in our code. But this field we don't want to show on UI instead we will show all the variables inside this class.